### ISSUE TITLE
The `verify_deprecated.h` header is not installed automatically

### ISSUE THREAD

**Issue by TomMD** [GuardTime/libksi#8]
When building and using libksi via the `rebuild-dep.sh` and `dpkg -i ...somepath/*.dpkg` I received the following error:

```
In file included from /usr/local/include/ksi/ksi.h:31:0,
                 from src/gt.c:1:
/usr/local/include/ksi/signature.h:25:31: fatal error: verify_deprecated.h: No such file or directory
 #include "verify_deprecated.h"
                               ^
compilation terminated.
```

Manually copying over the file to `.../include/ksi/` works.

**Comment by vii5ard** [comment]
Officially we do not support Debian packages. We recommend to use rpm packages instead. But we'll look into it.

**Comment by vii5ard** [comment]
Fixed in release [v3.10.1839](https://github.com/guardtime/libksi/releases/tag/v3.10.1839)
